### DISCUSSION TITLE
請問是否有一下子添加多個播客鏈接的功能呢?

### DISCUSSION THREAD

**Discussion by xjx0106** [yenche123/podcast-together#17] [想法]
請問是否有一下子添加多個播客鏈接的功能呢?

比如一次添加好多個節目進去，這樣就可以一次聼好久了... :)
因爲有些節目時間太短，每次都要重新創建一個房間，有點不方便，如果能直接添加節目到列表裏就好啦！！ :)

**Comment by yenche123** [comment]
會有點複雜，目前沒有計畫
不過感謝你的建議!
